docs/source/users/index.md

```diff
@@ -466,3 +466,56 @@ produced the following Python error:
 Write a new version of this code that does not produce that error.
 ```
 
+### Creating and managing aliases
+
+You can create an alias for a model using the `%ai register` command. For example, the command:
+
+```
+%ai register claude anthropic:claude-v1.2
+```
+
+will register the alias `claude` as pointing to the `anthropic` provider's `claude-v1.2` model. You can then use this alias as you would use any other model name:
+
+```
+%%ai claude
+Write a poem about C++.
+```
+
+You can also define a custom LangChain chain:
+
+```
+from langchain.chains import LLMChain
+from langchain.prompts import PromptTemplate
+from langchain.llms import OpenAI
+
+llm = OpenAI(temperature=0.9)
+prompt = PromptTemplate(
+    input_variables=["product"],
+    template="What is a good name for a company that makes {product}?",
+)
+chain = LLMChain(llm=llm, prompt=prompt)
+```
+
+… and then use `%ai register` to give it a name:
+
+```
+%ai register companyname chain
```

Comment:
Is there a non-interactive way to do this? It would be really tedious having to do so in every notebook (with alternative being using `%run` magic etc). Can we achieve the same via in an installable package, as it was originally proposed with cookiecutter-based modules?

Reply:
Thank you for bringing this up! Let's keep the conversation going in #190.